[16.0] [ADD] account_chart_update_multilang - new module (#1957)
OCA/account-financial-tools: account_chart_update_multilang/readme/DESCRIPTION.md

> This module extends the functionality of account_chart_update so that if the tax or fiscal position template has translated fields, it is passed to the final tax or fiscal position when applying the chart of accounts.

Reviewer: ```suggestion
This module extends the functionality of `account_chart_update` so that if taxes or fiscal position templates have translated fields, they are passed to the final taxes or fiscal positions when applying the chart of accounts.
```
